docs/topics/upgrade.md

```diff
@@ -38,7 +38,7 @@ In order to ensure that your `aks-engine upgrade` operation runs smoothly, there
 
 5) `aks-engine upgrade` is backwards compatible. If you deployed with `aks-engine` version `0.27.x`, you can run upgrade with version `0.29.y`. In fact, it is recommended that you use the latest available `aks-engine` version when running an upgrade operation. This will ensure that you get the latest available software and bug fixes in your upgraded cluster.
 
-### Under the hood
+### [Under the hood](#under-the-hood)
```

Comment:
I think this markdown link should be removed. It creates a hyperlink to the same `A HREF` that is on the same line (check the "View File" preview). If you just want to ensure there's an `HREF` for the headline, Markdown does that already.